test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -57,8 +57,26 @@ Describe 'Get-Help -Online opens the default web browser and navigates to the cm
                 [System.Management.Automation.Platform]::IsNanoServer
 
     It "Get-Help get-process -online" -skip:$skipTest {
-
-        { Get-Help get-process -online } | Should Not Throw
```

Comment:
Would it make more sense to check if the default browser is set? If it is, run the test; otherwise, skip it.